### ISSUE TITLE
icloudpd-1.17.6 keyring.errors.NoKeyringError: No recommended backend was available

### ISSUE THREAD

**Issue by cyayon** [icloud-photos-downloader/icloud_photos_downloader#841]
## Overview
Fatal error when using the last clouded-1.17.6
No issue with the previous version 1.17.5

## Steps to Reproduce
./icloudpd-1.17.6-linux-amd64 --username xxx@xxx --directory /xxxx --until-found 100

## Expected Behavior
no error

## Actual Behavior
```
2024-05-24 08:19:50 DEBUG    Authenticating...
Traceback (most recent call last):
  File "starters/icloudpd.py", line 6, in <module>
  File "click/core.py", line 1157, in __call__
  File "click/core.py", line 1078, in main
  File "click/core.py", line 1434, in invoke
  File "click/core.py", line 783, in invoke
  File "icloudpd/base.py", line 326, in main
  File "icloudpd/base.py", line 757, in core
  File "icloudpd/authentication.py", line 32, in authenticate_
  File "pyicloud_ipd/base.py", line 233, in __init__
  File "pyicloud_ipd/utils.py", line 34, in get_password_from_keyring
  File "keyring/core.py", line 55, in get_password
  File "keyring/backends/fail.py", line 25, in get_password
keyring.errors.NoKeyringError: No recommended backend was available. Install a recommended 3rd party backend package; or, install the keyrings.alt package if you want to use the non-recommended backends. See https://pypi.org/project/keyring for details.
``


## Context
Archlinux (last version)

**Comment by miroslav-suvada** [comment]
Getting the same error on Ubuntu 22.04. Running `keyring.get_password()` manually in python console works fine.
```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.4 LTS
Release:        22.04
Codename:       jammy
```

```
2024-05-24 10:28:16 DEBUG    Authenticating...
Traceback (most recent call last):
  File "starters/icloudpd.py", line 6, in <module>
  File "click/core.py", line 1157, in __call__
  File "click/core.py", line 1078, in main
  File "click/core.py", line 1434, in invoke
  File "click/core.py", line 783, in invoke
  File "icloudpd/base.py", line 326, in main
  File "icloudpd/base.py", line 757, in core
  File "icloudpd/authentication.py", line 32, in authenticate_
  File "pyicloud_ipd/base.py", line 233, in __init__
  File "pyicloud_ipd/utils.py", line 34, in get_password_from_keyring
  File "keyring/core.py", line 55, in get_password
  File "keyring/backends/fail.py", line 25, in get_password
keyring.errors.NoKeyringError: No recommended backend was available. Install a recommended 3rd party backend package; or, install the keyrings.alt package if you want to use the non-recommended backends. See https://pypi.org/project/keyring for details.
[24489] Failed to execute script 'icloudpd' due to unhandled exception!
```

**Comment by boredazfcuk** [comment]
Works on Alpine 3.20... which I've noticed isn't listed on the tests here: https://github.com/icloud-photos-downloader/icloud_photos_downloader/releases/tag/v1.17.7

**Comment by AndreyNikiforov** [comment]
fixed in 1.17.7. pls reopen if issue persists
